packages/blocks/package.json

```diff
@@ -24,21 +24,21 @@
   "dependencies": {
     "@blocksuite/global": "workspace:*",
     "@blocksuite/virgo": "workspace:*",
-    "@floating-ui/dom": "^1.5.1",
+    "@floating-ui/dom": "^1.5.3",
     "@types/webfontloader": "^1.6.35",
     "buffer": "^6.0.3",
     "date-fns": "^2.30.0",
-    "file-type": "^16.5.4",
+    "file-type": "^18.5.0",
+    "fractional-indexing": "^3.2.0",
     "html2canvas": "^1.4.1",
     "jszip": "^3.10.1",
-    "lit": "^2.8.0",
-    "marked": "^4.3.0",
+    "lit": "^3.0.0",
+    "marked": "^9.1.2",
```

Comment:
According to the CI log, there are too much breaking change here, and we have no plan to maintain the legacy related `ContentParser`. So not bumping this should be better.